### PR TITLE
Expedition Fluff Expansion - Stargazing & Fossil Stands

### PR DESCRIPTION
**Pull request by 1Sparky1**
Adds a stargazing mechanic, allowing you to lie down outside and see constellations. Skrell are naturally better at spotting constellations than other species. Telescopes can be used to see all the constellations visible, rather than a random mixture.

Adds fossil stands, in two variants. The smaller can take shell and plant fossils, while the larger takes multiple bones and a skull to create a display.

### CODE REVIEW

**Comment by 1Sparky1** [comment]
!review

**Comment by 1Sparky1** [comment]
!wip

File: code/datums/constellation.dm [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+	name = "The Island"
+	description = "A lonely island in the vacuum of space."
+
+/datum/constellation/egg
+	name = "The Hatching Egg"
+	description = "Is this how stars are born?"
+
+/datum/constellation/chanter
+	name = "The Star Chanter"
+	description = "IA worshiper praying to the stars they inhabit."
```
Comment (DreamySkrell) [inline code/datums/constellation.dm:71]: IA?

File: code/modules/overmap/_defines.dm [hunk 2]
```diff
@@ -15,6 +15,9 @@ GLOBAL_LIST_EMPTY(map_sectors)
 	icon_state = "map"
 	permit_ao = FALSE
 
+	//Stargazing stuff
+	var/list/visible_constellations = list()
```
Comment (DreamySkrell) [inline code/modules/overmap/_defines.dm:19]: document what is this list supposed to hold (types?)

**Comment by 1Sparky1** [comment]
!review

**Review by Arrow768 — COMMENTED** [review]
I am unsure if we should give the constellations "fixed" names.

For example "The Archer" implies that this is a very specific constellation.
But my understanding is that a specific constellation can only be seen from a specific star system. (As the angle/distance to the other stars changes when you change star systems; correct me if I am wrong please)

I think it would be better to introduce random components to the name to signify that it's not ""the"" constellation they are looking at.
(Possibly by using variables and a random word list per variable that's set on a constellation level; I.e. ('"%v1 %v2 archer" var/list/namevars = list("v1" = list("great", "lesser"), "v2" = list("small", "shortbow"))')

In addition I think that the constellations should be sector specific, as species would tend to name constellations after things that are important to them (unless no constellations would exist for that sector otherwise)

**Comment by 1Sparky1** [comment]
!wip

**Pull request closed without merging**